nucliadb/src/migrations/0039_backfill_converation_splits_metadata.py

```diff
@@ -102,5 +102,5 @@ async def build_splits_metadata(
         if page is None:
             continue
         for message in page.messages:
-            splits_metadata.metadata.setdefault(message.ident, SplitMetadata())
+            splits_metadata.metadata.get_or_create(message.ident)
```

Comment:
Changed in protobuf 6. `setdefault` cannot be used to set Messages (like SplitMetadata). They recommend `get_or_create` instead which always creates an empty/default message of the correct type.